daemon/proto/daemon.proto

```diff
@@ -29,6 +33,22 @@ message Team {
   string Id = 1;
 }
 
+message DestroyUserRequest {
+  string username = 1;
+}
+
+message DestroyUserResponse {
+  string message = 1;
```

Comment:
Add error parameter as well. we have to handle the response on the webclient.

Reply:
Message is used for both success or error  which ever comes first.

Reply:
it's better to have error message anyway. i think easier to manage in the webclient.

Reply:
@gianmarcomennecozzi, I am having difficulties to understand apparently, could you briefly explain me what would be the advantage of it ? How it will differ in webclient side to have this one ; 
```
message DestroyUserResponse {
  string message = 1; 
  string err = 2; 
}
```
instead 

```
message DestroyUserResponse {
  string message = 1;
}
```
or does it break some rules that we defined earlier and I do no remember now ?

Reply:
the advantage of it is on the webclient. when i parse the response on the webclient how can i understand if the response is ok or not ?
if we have `message` and `err` parameters its easy to check whatever we got an error from haaukins or not.

Reply:
thats why in `login` and `signup` response messages we have that `error` parameter in the proto file

Reply:
Ok, normally, I do not prefer to have it, however, I do not want to keep you annoyed :) as well. However, I guess, only difference will be just checking message instead of error, 

Example

```
 daemonclient.loginUser(getRequest, {}, (err, response) => {
         if (err == null && response.getError() == '') {
                      localStorage.setItem('user', response.getToken());
                      localStorage.setItem('user-email', username);
                      router.push(this.returnUrl)
          }else{
               this.error = err || response.getError();
               this.submitted = false;
               this.loading = false;
            }
    });

```
into this, only difference will be ` this.message = err || response.getMessage();`

```

 daemonclient.loginUser(getRequest, {}, (err, response) => {
         if (err == null && response.getError() == '') {
                      localStorage.setItem('user', response.getToken());
                      localStorage.setItem('user-email', username);
                      router.push(this.returnUrl)
          }else{
               this.message = err || response.getMessage();
               this.submitted = false;
               this.loading = false;
            }
    });

```

Ok, I am going to change it into error, I do not want to create a barrier in front of you @gianmarcomennecozzi  :)

Reply:
ohh wait. i miss some pieces of code. i have seen u were returning error as well. than u dont have to change it

Reply:
Yeah, that's what I was trying to express actually @gianmarcomennecozzi.

Reply:
@gianmarcomennecozzi, but keep in mind that you need to check content of the error message, because I am sending error as message and `success` as message in some cases. Like here: https://github.com/aau-network-security/haaukins/blob/7bfebccfc6f049853b1dfc2d6e906d492d536858/daemon/user.go#L176